connector_onpay/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_onpay_pay_type,onpay.pay.type,model_onpay_pay_type,base.group_user,1,1,1,1
+access_onpay_pay_type_group,onpay.pay.type,model_onpay_pay_type,group_onpay_allow_export,1,1,1,1
```

Comment:
I'm quite sure this access line is unnecessary.

Reply:
`group_onpay_allow_export` is a custom group introduced in this module. So I added that group. It is mentioned in the design.